### DISCUSSION TITLE
Last version for android.

### DISCUSSION THREAD

**Discussion by DavKiller** [ppy/osu#21538] [Q&A] [answered]
Hello everyone! 
I am trying to download the APK to update to the new version of Osu Lazer but I see that the APK is not there, since I don't know where to communicate it, I am writing it here.
The old version with the apk option
![Screenshot_2022-12-06-21-01-44-595_com.github.android.jpg](https://user-images.githubusercontent.com/65242580/206011441-210f0a90-5cdb-4390-b5e2-75c4f8a87aa1.jpg)
The new version without the apk option
![Screenshot_2022-12-06-21-01-33-148_com.github.android.jpg](https://user-images.githubusercontent.com/65242580/206011605-8f42babd-a006-4f67-badb-d58743959195.jpg)

**Comment by peppy** [comment, marked as answer]
There was an issue with the release. We'll publish a new one later today.
